Wangscape/TileGenerator.cpp

```diff
@@ -33,7 +32,7 @@ void TileGenerator::generate(sf::RenderTexture& image, size_t x, size_t y,
         sf::Sprite tile(temp.getTexture());
         tile.setPosition(sf::Vector2f((float)x, (float)y)*(float)options.resolution);
         image.draw(tile, sf::RenderStates(sf::BlendMode(sf::BlendMode::SrcAlpha, sf::BlendMode::One, sf::BlendMode::Add,
-                                                        sf::BlendMode::Zero, sf::BlendMode::One, sf::BlendMode::Add)));
+                                                        sf::BlendMode::Zero,     sf::BlendMode::One, sf::BlendMode::Add)));
```

Comment:
The extra 4 spaces are there to align the two lines. Arguments 1-3 and arguments 4-6 are symmetric in this constructor for `sf::BlendMode`. ([link](http://www.sfml-dev.org/documentation/2.4.1/structsf_1_1BlendMode.php#a69a12c596114e77126616e7e0f7d798b))